source/utils/security.py

```diff
@@ -26,17 +26,16 @@ def __getattr__(attrName: str) -> Any:
 	"""Module level `__getattr__` used to preserve backward compatibility.
 	"""
 	import NVDAState
-	if not NVDAState._allowDeprecatedAPI():
-		log.debug(f"Deprecated {attrName} imported while _allowDeprecatedAPI is False")
-	elif attrName == "isObjectAboveLockScreen":
-		log.warning(
-			"Importing isObjectAboveLockScreen(obj) is deprecated. "
-			"Instead use obj.isBelowLockScreen. "
-		)
-		return _isObjectAboveLockScreen
-	elif attrName == "postSessionLockStateChanged":
-		log.warning("postSessionLockStateChanged is deprecated, use post_sessionLockStateChanged instead.")
-		return post_sessionLockStateChanged
+	if NVDAState._allowDeprecatedAPI():
+		if attrName == "isObjectAboveLockScreen":
+			log.warning(
+				"Importing isObjectAboveLockScreen(obj) is deprecated. "

```

Comment:
While we are changing this, I think this wording makes more sense

```suggestion
				"isObjectAboveLockScreen(obj) is deprecated. "
```